index.Rmd

```diff
@@ -4,10 +4,10 @@ author: Joe Prego
  
 ---
 
-# Introduction
+# Introduction 
```

Comment:
One suggestion:  put language in the present rather than the future tense ("this project examines" vs "this project will examine...").  Probably a holdover from the proposal stage.  But worth revisiting throughout the project, I think.